app/src/ui/lib/conflicts/unmerged-file.tsx

```diff
@@ -122,7 +122,7 @@ const renderResolvedFile: React.SFC<{
     <li key={props.path} className="unmerged-file-status-resolved">
       <Octicon symbol={OcticonSymbol.fileCode} className="file-octicon" />
       <div className="column-left">
-        <PathText path={props.path} availableWidth={200} />
```

Comment:
Due to “Open in \<EDITOR>”  buttons on the right side, which are of variable width depending on which editor was selected, we don't really know how much space the PathText component has available. Therefore, we leave it to be calculated instead.